### DISCUSSION TITLE
Missing background image when loading the player in IOS

### DISCUSSION THREAD

**Discussion by tommy9690** [vimeo/player.js#1016] [Bug reports]
### Expected Behavior
It should have the initial thumbnail image when the player is loaded in IOS

### Actual Behavior
When the player is loaded in IOS, the player only show the control with transparent background

<img width="291" alt="image" src="https://user-images.githubusercontent.com/6504495/224563308-546d9f59-dd05-464c-a77a-0e88ddea2ed5.png">

**Comment by Marcoo09** [comment]
I have the same problem using https://github.com/MetaLabs-inc/react-native-vimeo-iframe
It started to happen on IOS 16.

**Comment by rkrishnan8594** [comment]
@tommy9690 @Marcoo09 Can you provide a reproducible example of this issue?

**Comment by Marcoo09** [comment]
Hi @rkrishnan8594, thanks for your answer.
If you open [this](https://player.vimeo.com/api/demo) demo in the browser or an app you'll see the problem.
<img width="399" alt="image" src="https://user-images.githubusercontent.com/18408823/229568095-eb2f7ad7-29f0-4956-9eb3-d3ed7d65454c.png">

**Comment by rkrishnan8594** [comment]
@Marcoo09 I'm not able to reproduce this issue on Safari iOS 16.3.1. What version are you using to reproduce?

**Comment by Marcoo09** [comment]
@rkrishnan8594 are you testin on an Iphone with IOS 16? I'm testing with 16.3.1 too.

**Comment by rkrishnan8594** [comment]
I'm testing with an iPhone 12.

**Comment by Marcoo09** [comment]
@rkrishnan8594 I'm talking about the OS version

**Comment by rkrishnan8594** [comment]
Yes, iOS 16.3.1.

**Comment by bdougherty** [comment]
@tommy9690 @Marcoo09 are you still seeing the issue?
